tests/gpt.rs

```diff
@@ -120,14 +120,50 @@ fn test_create_simple_on_device() {
     gdisk.update_partitions(BTreeMap::<u32, gpt::partition::Partition>::new()).unwrap();
     // At this point, gdisk.primary_header() and gdisk.backup_header() are populated...
     // Add a few partitions to demonstrate how...
-    gdisk.add_partition("test1", 1024 * 12, gpt::partition_types::BASIC, 0).unwrap();
-    gdisk.add_partition("test2", 1024 * 18, gpt::partition_types::LINUX_FS, 0).unwrap();
+    gdisk.add_partition("test1", 1024 * 12, gpt::partition_types::BASIC, 0, None).unwrap();
+    gdisk.add_partition("test2", 1024 * 18, gpt::partition_types::LINUX_FS, 0, None).unwrap();
     let mut mem_device = gdisk.write().unwrap();
     mem_device.seek(std::io::SeekFrom::Start(0)).unwrap();
     let mut final_bytes = vec![0u8; TOTAL_BYTES];
     mem_device.read_exact(&mut final_bytes).unwrap();
 }
 
+fn test_create_aligned_on_device() {
```

Comment:
This test get's never executed.
And if the test get's enabled the following error occurs:
```
thread 'test_create_aligned_on_device' panicked at 'unexpected error writing second aligned partition: should start at LBA 56, end at 61', tests\gpt.rs:159:5
```

Reply:
Thanks for the heads up. Will take a look. Might make sense to revert in the short term? Traveling a bit this weekend but will have downtime on Sunday to take a look.

Reply:
Since its already been in the codebase for ~2 years I think we can wait until you have looked at it. Just open a Pr or an issue with you findings.